tpcx_bb/benchmark_runner.py

```diff
@@ -1,45 +1,93 @@
+import glob
+import re
 import os
 import gc
 import time
 
+
 os.environ["tpcxbb_benchmark_sweep_run"] = "True"
 
-qnums = [str(i).zfill(2) for i in range(1, 31)]
-n_repeats = 3
+N_REPEATS = 1
+
+
+def get_qnum_from_filename(name):
```

Comment:
Not necessary to do in this PR, but we should include the `load_test` script somehow.

Reply:
Sounds good. Will flag for follow up